appsec/_test/appsec_test.go

```diff
@@ -0,0 +1,67 @@
+package _test_test
```

Comment:
It is allowed in Go to have test files in a package under a different package name for the sake of tests. You can move the contents of this folder back into `appsec` and name the package `appsec_test`
```suggestion
package appsec_test
```
This is the [idiomatic](https://go-review.googlesource.com/c/go/+/145279/) way in Go. We don't need the `_test` package. Packages starting with underline (presumably trying to signify an internal or private package) are not used in Go.

Reply:
That's actually to avoid having its dependencies in the appsec go.mod file. There a new one in _test so that I can write this validation test that imports the tracer. Otherwise, the tracer would end up in the appsec go.mod file

Reply:
The AppSec module and this one are now removed by 2259fa6dd2c9ad6c2a310125f68a2e91a0e011df
But I moved this test into waf_test.go as it is very appsec-specific, so it felt wrong to me to have it in `contrib/net/http`. WDYT?

Reply:
Fine by me. Thanks.